assets/js/components/ClusterDetails/ChecksSelection.jsx

```diff
@@ -101,6 +101,14 @@ function ChecksSelection({ clusterId, cluster }) {
     }
   }, [loading]);
 
+  const saveSelection = () =>
+    dispatch(
+      checksSelected({
```

Comment:
Wouldn't be better to pass `selectedChecks` and `clusterId` as the function arguments?
Using global args looks a bit unsafe.
But talking from my ignorance... XD

Reply:
We would need to have different names tho, otherwise it will clash.
And at the end we would be still passing the same variables, with a different name.

I am open to suggestions.

Reply:
I don't really have a suggestion...
Anyway, now that i checked a bit, we should maybe wrap this functions in `useCallback`, otherwise the function is redefined in every render. Maybe a cheap operation, but with the `useCallback` it is avoided.
Maybe @dottorblaster has stronger opinions on this